src/einsteinpy/utils/schwarzschild_utils.py

```diff
@@ -1,153 +0,0 @@
-import astropy.units as u
-import numpy as np
-
-from einsteinpy import constant
-
-
-def schwarzschild_radius(mass, c=constant.c, G=constant.G):
-    """
-    Schwarzschild radius is the radius defining the event horizon of a
-    Schwarzschild black hole. It is characteristic radius associated with every
-    quantity of mass.
-
-    Parameters
-    ----------
-    mass : ~astropy.units.kg
-
-    Returns
-    -------
-    r : ~astropy.units.m
-        Schwarzschild radius for a given mass
-
-    """
-    if not isinstance(mass, u.quantity.Quantity):
-        mass = mass * u.kg
-    if not isinstance(c, u.quantity.Quantity):
-        c = c * u.m / u.s
-    if not isinstance(G, u.quantity.Quantity):
-        G = G * constant.G.unit
-    M = mass.to(u.kg)
-    num = 2 * G * M
-    deno = c ** 2
-    return num / deno
-
-
-def schwarzschild_radius_dimensionless(M, c=constant.c.value, G=constant.G.value):
-    """
-    Parameters
-    ------------
-    M : float
-        Mass of massive body
-    c : float
-        Speed of light, defaults to value of speed of light in SI units.
-    G : float
-        Gravitational Constant, defaults to its value in SI units
-    Returns
-    -------
-    Rs : float
-        Schwarzschild radius for a given mass
-    """
-    Rs = 2 * M * G / c ** 2
-    return Rs
-
-
-@u.quantity_input(mass=u.kg)
-def time_velocity(pos_vec, vel_vec, mass):
-    """
-    Velocity of time calculated from einstein's equation.
-    See http://www.physics.usu.edu/Wheeler/GenRel/Lectures/GRNotesDecSchwarzschildGeodesicsPost.pdf
-
-    Parameters
-    ----------
-    pos_vector : ~numpy.array
-        Vector with r, theta, phi components in SI units
-    vel_vector : ~numpy.array
-        Vector with velocities of r, theta, phi components in SI units
-    mass : ~astropy.units.kg
-        Mass of the body
-
-    Returns
-    -------
-    ~astropy.units.one
-        Velocity of time
-
-    """
-    # this function considers SI units only
-    a = schwarzschild_radius(mass).value
-    c = constant.c.value
```

Comment:
I was trying to find where did you do the change of the issue #507 but I wasn't able to find it. Can you point me where it is, please? Thank you!

Reply:
`time_velocity` methods, across `<Metric>_utils` modules, have been [moved to and repackaged into 1 function in `coordinates.utils`](https://github.com/JeS24/einsteinpy/blob/0f90cb36d9f90e51659f3ddf204e6cf0302121c4/src/einsteinpy/coordinates/utils.py#L209-L250), renamed as `v_t()`. This function uses the supplied metric and 3-Velocity, and the expression for spacetime interval: $g_{ab} u^a u ^b = 0 or 1$ (null-like or time-like), to determine the timelike component of 4-Velocity, and hence it bypasses the method used here.

Also, my bad. I should put this in the PR description.

Reply:
Also, I should note, that the method used here is the same, only difference being that this hardcodes the metric components.